src/main/controller/UserController.ts

```diff
@@ -0,0 +1,43 @@
+import { Request, Response } from 'express';
+import db from '../models';
+import { Route } from '../constant/Route';
+
+export class UserController {
+  getUser = async (req: Request, res: Response) => {
+    try {
+      if (!req.params.userId) {
+        throw `Missing userId param. Usage: /${Route.USERS}/:userId`;
```

Comment:
Same as before but with error throwing! If we can define an Error class that contains all the various errors that we could throw- that would be great!